blog/_src/posts/2016-12-17-measuring-the-submission-review-balance.md

```diff
@@ -0,0 +1,84 @@
+    Date: 2016-12-17T16:33:10
+    Tags: by Gabriel Scherer
+
+How do researchers know whether they are doing "enough" or "too many"
+reviews? A measurable goal is to be review-neutral: to have demanded,
+through our submissions, as many reviews as we have produced as
+reviewers.
+
+<!-- more -->
+
+### Reviewing is good
+
+I like to review academic papers. It is a very rewarding activity in
+many different ways. One gets to serve the academic community, helping
+it function smoothly. One gets a chance at acquiring a much better
+understanding of someone else's work than idle paper-skimming
+allows. One get to send feedback to our colleagues and help them
+improve their work and its presentation -- it is also an essential way
+in which we can participate to the formation of student researchers
+all over the world. Finally, doing reviews helped me develop the skill
+the judge someone else's work and of forcing oneself to come with
+a decisive opinion -- it is surprisingly difficult and only comes with
+training.
+
+Doing reviews is also fairly time-consuming. I noticed that the time
+I spend on each review is generally stable (excursions into previous
+or related work excluded): around one day and a half for conference
+reviews, and at least twice more for journal reviews -- I'm sure other
+people have wildly different figures, but I would expect it to be
+a noticeable time commitment in any case. (Workshop reviews are much
+easier, at least for the formats I have seen of 2-page extended
+abstracts, I'd say one hour per review.)
+
+### How many reviews?
+
+Because it is so time-consuming, deciding whether to say "yes" or "no"
+to invitations to review a new paper is not easy: in general I want to
+say "yes" (unless I can tell that I will not enjoy reading the paper
+at all), but it is not reasonable to say "yes" all the time, because
+I also need to spend time on other things. When should I say "no"
+because I have done "too much" reviews already?
+
+We can count the number of reviews that we have done, and we can also
+estimate the number of reviews that we have demanded of other through
+our submissions. A natural goal for researchers is to produce at least
+as many reviews as they demand; if everyone reached this goal, the
+peer-review system would be at equilibrium without imposing too much
+of a workload on anyone.
+
+To estimate the number of reviews a researcher demanded from their
+peers, you can sum, for each of their submissions to a peer-reviewed
+venue, the number of reviews that they received, divided by the total
+number of authors of the submissions.
+
+\\[ \sum_{p \in \mathtt{Submissions}} \frac{\mathtt{reviews}(p)}{\mathtt{authors}(p)} \\]
+
+Out of curiosity, I just measured this balance for myself: so far
+I have "demanded" 13 workshop reviews and 29 conference reviews, and
+"produced" 6 workshop reviews and 17 conference reviews. If you think
+that an article would interest me, you shouldn't feel bad about asking
```

Comment:
The 13 and 29 numbers come from the formula right? But surely you must have rounded to get whole numbers?

If that's the case I would suggest not rounding bc it confused me at first glance since you would expect decimals so I thought it might be a different measure.